tests/dts/api.test-d.ts

```diff
@@ -64,11 +65,14 @@ expectType<(callback: (error: Error | string) => boolean | { group: string; }) =
 expectType<(name: string, id: string) => any>(microAgent.addRelease)
 expectType<(message: string, options?: { customAttributes?: object, level?: 'ERROR' | 'TRACE' | 'DEBUG' | 'INFO' | 'WARN'}) => any>(microAgent.log)
 
-// The following browser agent APIs should not be available in the Micro Agent:
-expectError(() => expectType<any>(microAgent.addToTrace))
-expectError(() => expectType<any>(microAgent.setCurrentRouteName))
-expectError(() => expectType<any>(microAgent.interaction))
-expectError(() => expectType<any>(microAgent.finished))
-expectError(() => expectType<any>(microAgent.recordReplay))
-expectError(() => expectType<any>(microAgent.pauseReplay))
-expectError(() => expectType<any>(microAgent.wrapLogger))
```

Comment:
The microagent type checks shouldn't be removed until microagent itself is actually removed

Reply:
b98a47fe